e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -1242,31 +1270,71 @@ describe("scenarios > embedding > full app", () => {
         });
       });
 
-      it("should be able to join a card when the data source is a table", () => {
-        const cardDetails = {
-          ...ordersCardDetails,
+      it("should join a table when the data source is a model", () => {
+        // Orders Model already exists
+        const ordersModelName = "Orders Model";
+        const ordersCountModelDetails = {
+          ...ordersCountCardDetails,
+          name: "Orders Count Model",
           type: "model",
-          collection_id: FIRST_COLLECTION_ID,
+          collection_id: null,
```

Comment:
Just curious why we changed from the first collection to Our analytics here

Reply:
We did not change anything here as per my [comment above](https://github.com/metabase/metabase/pull/57744/files/2d68693dbdd88f370172b595da266955806c9ebf#r2081788404). The diff is just being funny. This is a completely new test.

The test name isn't even the same as the one before 😅

Reply:
ah! I thought using split diffs would making it clearer, turns out it's still confusing 😆